ship-show-ask/action.yml

```diff
@@ -13,4 +14,15 @@ runs:
           show 👀
           ask ❓
         add_comment: true
-        message: "Denne PR-en kan ikke merges før du merker den med en av lablene: {{ provided }}"
+        message: "Denne PR-en mot ${{ github.base_ref }} kan ikke merges før du merker den med en av lablene: {{ provided }}"
+    - if: github.base_ref != 'main'
+      uses: mheap/github-action-required-labels@v5
+      with:
+        mode: exactly
+        count: 0
+        labels: |
+          ship 🚀
+          show 👀
+          ask ❓
+        add_comment: true
+        message: "Denne PR-en mot ${{ github.base_ref }} skal derfor ikke ha noen av disse lablene: {{ provided }}"
```

Comment:
Mulig jeg ikke ser for meg riktig åssen det blir men kanskje "derfor" blir litt forvirrende her?

Reply:
Meget godt poeng, den setningen ga ikke mening! Omformulerte i 81c1f7f57488857e66fc5bc0c5ddc6d08467d619